applications/server/modules/access.py

```diff
@@ -0,0 +1,83 @@
+from gluon import current
+from gluon import auth
+
+
+# Permission types.
+# v = view
+# a = admin (valid only for one whole device)
+# e = edit settings of procedure
+
+def register_device(name=None, description=''):
+    db = current.db
+    devices = db.devices
+    permission = db.user_permission
+    device_id = devices.insert(name=name, user_id=auth.user_id, description=description)
+    permission.insert(perm_user_email=auth.user.email, device_id=device_id, procedure_id=None, perm_type='a')
+
+
+# Share one device to multiple users with one permission type
```

Comment:
Use docstring comments:

def f(x):
    """This function computes something.
       @param x : the floating point value to process
       @returns : the result of the computation.
   """